cleanlab/classification.py

```diff
@@ -113,6 +113,7 @@ def score(self, X, y, sample_weight=None):
 the :math:`\\tilde{y}` used in confident learning paper.
 """
 
+from cleanlab.experimental.label_issues_batched import find_label_issues_batched
```

Comment:
[nit] I would move this import a little lower (perhaps line 152) to group it together with the rest of the cleanlab imports - makes it a little neater